config-service-impl/src/main/java/org/hypertrace/config/service/ConfigServiceUtils.java

```diff
@@ -59,4 +59,8 @@ public static Value merge(Value defaultConfig, Value overridingConfig) {
   public static String getActualContext(String rawContext) {
     return Strings.isNullOrEmpty(rawContext) ? DEFAULT_CONTEXT : rawContext;
   }
+
+  private static boolean isNull(Value value) {
+    return value == null || value.getKindCase() == Value.KindCase.NULL_VALUE;
```

Comment:
It's fine to be defensive, but out of curiosity - is it ever possible for value to actually be `null`? A proto should never  return null, that's one of their design principles, so any nulls we're encountering would be from our own code and we should consider a bug. Given that, should we log an error in the `null` case and convert it to a `NULL_VALUE`?

Reply:
Yeah, value should not be `null`. Updated it to log an error on seeing a `null` value and convert it to `NULL_VALUE` wherever required.